Paintroid/src/main/res/values/string.xml

```diff
@@ -143,11 +145,21 @@
     <string name="dialog_loading_image_failed_text">Not a valid image</string>
     <string name="dialog_settings">Settings</string>
 
+    <string name="dialog_save_image_name">Image name</string>
+    <string name="dialog_save_image_format">Image format</string>
+
+    <string name="dialog_save_jpg_option_quality">Quality:</string>
+    <string name="dialog_save_options_size">Size:</string>
```

Comment:
Where is this string used?

Reply:
in dialog_pocketpaint_save.xml and dialog_pocketpaint_save_jpg.sub_dialog.xml .